### ISSUE TITLE
Change test framework to weaver

### ISSUE THREAD

**Issue by lukestephenson-zendesk** [realestate-com-au/applied-scala#38]
I'd like to change the test framework to weaver so this project hightlights scala at its best https://disneystreaming.github.io/weaver-test/.  I don't think specs2 does that. Thoughts?

**Comment by george-wilson-rea** [comment]
Agreed, weaver seems like a better choice these days, particularly when the project is using cats.
Please make it so :)
